### ISSUE TITLE
Update CMS Button audit to search additional fields

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15602]
## Description 
There is currently a [button audit](https://prod.cms.va.gov/admin/content/audit/buttons) in the CMS. As stated within the description, 

> This audit searches through all field_wysiwyg to find instances of 'usa-button' within the content which is used to create buttons.

However, we are intending to use this audit to help us clean up all the instances of fake buttons that never should have been created in the first place and this audit is too limited for this use. Prior to the D10 update, editors were able to create "buttons" in any rich text field, including fields nested within paragraphs. These nested fields are not being surfaced in the audit. 

Would it be possible to update the audit to search all these fields:
- Rich text (pretty sure this is the one field it's already searching)
- Accordion Item
- Expandable Text
- Additional information
- Rich text - char limit 1000
- Number callout
- Facility location support service
- Situation update
- Step

## Acceptance Criteria
- [ ] CMS Button Audit is updated to include all instances of `field_wysiwyg`
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [x] CMS Team
- [ ] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing
